conf/plants/mlss_conf.xml

```diff
@@ -383,6 +640,10 @@
       homology_range_index="27"
       prefer_for_genomes="hordeum_vulgare"
     />
+    <protein_trees
+      collection="oat_cultivars"
+      homology_range_index="40"
```

Comment:
I feel this is a little on the high side, @JAlvarezJarreta, as the way the `homology_range_index` is used to offset gene tree- and homology-related tables (in [lines 55-59 of the OffsetTables runnable](https://github.com/Ensembl/ensembl-compara/blob/93862cf7d4ff1e9169bdb8abfc668992877dff59/modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/OffsetTables.pm#L55-L59)) could leave us with less headroom than we might like in tables such as `gene_tree_node`.

That said, I'm wary of suggesting a specific number here without the opportunity to seek the input of @Simarpreet-Kaur-Bhurji  and @jitender-ebi .

Reply:
Happy to change it for the value you suggest. Just went for a random value that felt "right" based on the others, without having a clue what I was doing 😅 But it has helped me learn something new (or most likely remember something most likely you already shared), so I get something back 😁

Reply:
Thanks @JAlvarezJarreta. Following some discussion, I've suggested the number 34. This should leave plenty of space in either direction.